ch12/linear_models.md

```diff
@@ -13,4 +13,4 @@ table or total bill—in order to make our model more useful?
 In this chapter we will introduce linear models which will allow us to make use
 of our entire dataset to make predictions. Linear models are not only widely
 used in practice but also have rich theoretical underpinnings that will allow
-us to understand future tools for modeling.
+us to understand future tools for modeling. We first introduce simple linear regression with one explanatory variable, apply gradient descent to the linear model, and extend to multiple linear regression with more variables but the same underlying concepts.
```

Comment:
"apply gradient descent to the linear model" is a bit imprecise; we apply gradient descent to fit a model, not to the model itself.

"multiple linear regression with more variables but the same underlying concepts" has redundant pieces—"multiple" implies "more variables", "linear regression" implies "same underlying concepts". How about:

> We introduce a simple linear regression model that uses one explanatory variable, explain how gradient descent is used to fit the model, and finally extend the model to incorporate multiple explanatory variables.